src/sentry/integrations/bitbucket/webhook.py

```diff
@@ -104,9 +108,55 @@ def __call__(self, organization, event):
                     pass
 
 
+class PullEventWebhook(Webhook):
+    def __call__(self, organization, event):
+        repo = self.repo_from_event(organization, event)
+        pull_request = event['pullrequest']
+
+        integration = Integration.objects.get(
+            id=repo.integration_id,
+        )
+        installation = integration.get_installation(organization_id=organization.id)
+        emails = installation.get_client().get_user_emails()['values']
+        try:
+            commit_author = CommitAuthor.objects.filter(
+                organization_id=organization.id,
+                email__in=[email['email'] for email in emails if email['is_confirmed']],
+            )[0]
```

Comment:
Can there be more than one of these?

Reply:
Yes. Since I am filtering based on a list of email addresses rather than a single email address it could potentially come up with more than one commit_author. If the user is represented as a `CommitAuthor` in with two different entries somehow. I'm not sure how typical it is.

Reply:
@mnoble I understand but unfortunately that is inconsistent with how commitauthor typically has worked in the past. @macqueen you can let me know if I'm mistaken, but my understanding here is that users are matched via email right now, and it'll have to be consistent with what we have.

Reply:
Ok @mnoble I don't know a ton about how commit code works (so you can feel free to give feedback here), but https://github.com/getsentry/sentry/blob/master/src/sentry/models/commitauthor.py#L23 this method is used to get users in 
https://github.com/getsentry/sentry/blob/f5e916b28f9c4d0dad758c2493d7a3121465251c/src/sentry/receivers/releases.py#L52
and 
https://github.com/getsentry/sentry/blob/a8614be6018831efe6155b3768c59c7aa171db89/src/sentry/models/release.py#L508

So, actually I'm not sure how else you would do this without creating some changes to how the find_users method works. What do you think on the feasibility of that?